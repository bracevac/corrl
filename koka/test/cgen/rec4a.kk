function foo(xs,n) {
    function bar(xs) {
       match(xs) {
         Cons(_x,xx) -> bar(xx);
         Nil -> n
       }
    };
    bar(xs)
}


function main() { print(foo([],1)) }
