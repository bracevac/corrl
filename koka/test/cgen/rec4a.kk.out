1test/cgen/rec4a.kk(2,18): warning: xs shadows an earlier local definition or parameter